src/pages/Player.vue

```diff
@@ -44,14 +44,14 @@
 
 <script>
 import Plyr from "plyr";
-import PlioQuestion from "@/components/PlioQuestion.vue";
-import Error from "@/views/Error.vue";
-import LoadingSpinner from "@/components/LoadingSpinner.vue";
-import StartButtonPointer from "@/components/tutorial/StartButtonPointer.vue";
-import UserProperties from "@/components/UserProperties.vue";
-
-import PlioService from '@/services/PlioAPIService.js'
-import UserService from '@/services/UserAPIService.js'
+import PlioQuestion from "@/components/Items/PlioQuestion.vue";
+import Error from "@/pages/Error.vue";
+import LoadingSpinner from "@/components/UI/LoadingSpinner.vue";
+import StartButtonPointer from "@/components/UI/tutorial/StartButtonPointer.vue";
```

Comment:
`Tutorial` is part of `UI`? Why? `Tutorial` can be its own thing with its own `UI` folder, no?

Reply:
My thought was that tutorial components will be included in different pages, just like a delete button component (which is a UI component) will be included in different pages

Reply:
Haan, makes sense, I guess.

Use capital `T` for `Tutorial` :P